src/api/public/apidocs-new/paths/source_project_name_cmd_copy.yaml

```diff
@@ -0,0 +1,52 @@
+post:
+  summary: Copy the entire project.
+  description: |
+    Copy the entire project.
```

Comment:
What about replacing this description with something like you explained in the [PR description](https://github.com/openSUSE/open-build-service/pull/11912#issue-791624647)?

Otherwise, I would remove the description if it's the same as the summary.